### ISSUE TITLE
Create 0.1 branch

### ISSUE THREAD

**Issue by scarrazza** [qiboteam/qibolab_platforms_qrc#182]
We should mirror the release strategy from qibolab in order to avoid issues during installation.

**Comment by stavros11** [comment]
I have created the 0.1 branch based on the current main and updated the workflows to use qibolab 0.1 versions here: https://github.com/qiboteam/qibolab_platforms_qrc/tree/0.1. There are still two workflows: stable is using the latest qibolab release <0.2 from pypi and dev is using the qibolab 0.1 branch on github. The next step is to switch all open PRs containing 0.1 platforms to that branch instead of main. We may also want to disable direct pushing to that branch, similarly to main.

@alecandido @scarrazza shall I now remove all 0.1 platforms from main and merge qw11qD which is the only one working so far with 0.2?

**Comment by alecandido** [comment]
> @alecandido @scarrazza shall I now remove all 0.1 platforms from main and merge qw11qD which is the only one working so far with 0.2?

Yes, and I'm going to point all 0.1-based PRs to the `0.1` branch.

**Comment by scarrazza** [comment]
Yes, please go ahead.
